example/ameiva/falling_stars/helpers.dart

```diff
@@ -0,0 +1,49 @@
+import 'dart:math' as math;
+
+final math.Random _rnd = math.Random();
+int getRandomNumber(int min, int max) => min + _rnd.nextInt(max - min);
+
+class Vector {
+  num x, y;
+  Vector([this.x, this.y]);
+
+  String toString() => 'Vector($x, $y)';
+}
+
+class Point {
+  double x, y;
+  Point(this.x, this.y);
+
+  // Operator Overlay
+  // Allow us change an operato like ===, or, + , -
+  // Here we change the minus to allow it adde a Vector to the current point
+  // The x and y the will be adde to x and y to the point and will be returned a new point.
+  Point operator -(Vector vector) {
+    Point potision = Point(0, 0);
```

Comment:
Please, change "**potision**" to "_position_" in the whole document...

Reply:
I will make all that changes tomorrow.